tools/build.hpp

```diff
@@ -39,9 +44,16 @@ namespace antler {
          system::fs::create_directory(bin_dir);
          system::info_log("Building project...");
 
-         CLI::results_t args = {"--build", bin_dir.string(), "-j"};
-         if (jobs)
-            args.push_back(std::to_string(jobs));
+         CLI::results_t args = {"--build", bin_dir.string()};
+         if(jobs != std::numeric_limits<uint32_t>::max()) {

```

Comment:
so why did you decide to change default behavior for jobs?
currently if jobs are not specified we pass `-j` which corresponds for max available cores.
you changed it the way that by default you have always 1 job (`make` default behavior).
if you decided to be consistent with cmake, please update documentation.

Reply:
I feel like this provides the cleanest behavior. The default method works the same for any version of CMake.

Reply:
Updated help in 0d314e1.

If you have CMake set with a Ninja as the default generator, you will have multiple jobs.